doc/README.md

```diff
@@ -15,27 +24,18 @@ Or, if you'd rather not build the example gallery:
 
     make html-noplot
 
-That should create all the doc in directory _build/html
+That should create all the doc in directory `_build/html`
 
 To build the PDF manual, run
 
     make latexpdf
```

Comment:
Maybe add the [minimal latex requirements somewhere](https://github.com/scikit-learn/scikit-learn/blob/master/build_tools/circle/build_doc.sh#L91)? I know it's system/distribution dependent, but it could be useful to know what are the [bare minimum latex requirements](https://github.com/scikit-learn/scikit-learn/blob/master/build_tools/circle/build_doc.sh#L90-L94)  without having to install texlive-full (~6GB).  Maybe something similar to,
"""
To build the PDF manual, you need to install the requirements above as well as LaTex. For instance, on Debian or Ubuntu Linux,  you would need at least the following packages, 

       dvipng texlive-latex-base texlive-latex-extra texlive-latex-recommended
       texlive-latex-extra texlive-fonts-recommended latexmk

The PDF manual can be generated with,

    make latexpdf
"""
What do you think? Unless there is a sphinx documentation somewhere stating which LaTex packages are needed to build the pdf docs (I haven't seen one).

Reply:
Uhh... The Circle build script says this is actually for the HTML build, not latexpdf.

I'm ambivalent about adding this documentation, TBH. Perhaps it would be worth having an apt-requirements.txt and a conda-requirements.txt file in `doc/`.

Reply:
IMO this kind of very explicit doc gets outdated very quickly. I would be in favour of staying quite vague about the latex requirements and adding a link to build_doc.sh on github for the full details.

Also I don't think many people care about building the PDF locally. It is not very common that a PR breaks the pdf and not the html documentation.

Reply:
I agree about this being maybe too specific and os dependent. Also, it's true that not many people would need to build the pdf docs.